core/src/main/java/dk/alexandra/fresco/lib/lp/LPSolver.java

```diff
@@ -170,7 +187,8 @@ private boolean checkDimensions(LPTableau tableau, Matrix<DRes<SInt>> updateMatr
    *
    * @return a protocol producer for the first half of a simplex iteration
    */
-  private DRes<LpState> phaseOneProtocol(ProtocolBuilderNumeric builder, LpState state,
+  private DRes<Pair<List<DRes<SInt>>, DRes<BigInteger>>> phaseOneProtocol(
```

Comment:
The new return type is not reflected in the javadoc ... of course the old one wasn't either :(